tests/roots/test-latex-includegraphics/index.rst

```diff
@@ -0,0 +1,30 @@
+====================
+Test image inclusion
+====================
+
+Tests with both width and height
+--------------------------------
+
+.. an image with big dimensions, ratio H/W = 1/5
+.. image:: img.png
+   :height: 200
+   :width: 1000
+
+.. topic:: Oversized images
+
+   .. an image with big dimensions, ratio H/W = 5/1
+   .. image:: img.png
+      :height: 1000
+      :width: 200
+
+Tests with only width or height
+-------------------------------
+
+.. topic:: Oversized images
+
+   .. image:: img.png
+      :width: 1000
+
+.. image:: img.png
+   :height: 1000
```

Comment:
Due to the code in `conf.py` this test file must only test oversized images, so that `\spx@image@requiredheight` and `\spx@image@requiredwidth` as used in patched from conf.py  include graphics do actually refer to current image and not a previous one.